webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -197,9 +197,9 @@ const GroupMemberList = (props: Props) => {
                                     {user: name})}
                                 onClick={() => showDirectChannel(user)}
                             >
-                                <LocalizedIcon
+                                <i
                                     className='icon icon-send'
-                                    ariaLabel={{id: t('user_group_popover.close'), defaultMessage: 'Close'}}
+                                    aria-label={formatMessage({id: 'user_group_popover.close', defaultMessage: 'Close'})}
```

Comment:
Please remove this aria label because the parent button already have the aria label on it, so we don't need it here

Reply:
ok no problem